tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.Repository/SharePointInfo/SPContentTypes.cs

```diff
@@ -172,23 +172,26 @@ private void SetDefaultContentType(ClientContext clientContext, List list, Clien
                 fields.GetByInternalNameOrTitle(contentTypesConfig.ContentTypeColumnClientId).ReadOnlyField = true;
                 fields.GetByInternalNameOrTitle(contentTypesConfig.ContentTypeColumnClientId).SetShowInDisplayForm(true);
                 fields.GetByInternalNameOrTitle(contentTypesConfig.ContentTypeColumnClientId).Update();
-                fields.GetByInternalNameOrTitle(contentTypesConfig.ContentTypeColumnClientName).ReadOnlyField = true;
-                fields.GetByInternalNameOrTitle(contentTypesConfig.ContentTypeColumnClientName).SetShowInDisplayForm(true);
-                fields.GetByInternalNameOrTitle(contentTypesConfig.ContentTypeColumnClientName).DefaultValue = matterMetadata.Client.Name;
-                fields.GetByInternalNameOrTitle(contentTypesConfig.ContentTypeColumnClientName).Update();
+                if(configuration.GetSection("General")["IsBackwardCompatible"].ToString()=="false")

```

Comment:
If IsBackwardCompatible, then dont add client name as default value to the document library, Add managed column called Team Name as default value to the document library